.github/workflows/main.yml

```diff
@@ -101,6 +101,13 @@ jobs:
       - uses: ./.github/actions/rustup
       - uses: ./.github/actions/sccache
       - run: RISC0_SKIP_BUILD=1 cargo doc --no-deps --exclude=risc0-zkvm-methods --workspace
+      - name: Install mdbook
+        uses: risc0/cargo-install@2cfb0024938d23011106cbf127b393bc83fddba1
+        with:
+          crate: mdbook
+          version: "0.4"
+      - run: mdbook test
```

Comment:
This looks good, might be good to add a single test to verify its working as expected.

We could look into auto publishing (on main or on-release workflows): https://github.com/peaceiris/actions-mdbook Because we already have a GH token to publish to the `benchmarks` repo which has a public HTML page: https://risc0.github.io/benchmarks/

Reply:
Agreed and then we could probably just rename `benchmarks` to something more generic, "assets", "internal-website", ???.

Reply:
Agreed, it got overloaded when I started pushing crate-validator results there. I had it on my TODO to rename that repo but required changing a few other things. I can probably do that sooner though if we land on a name.  `generated-content` might work?

Reply:
The only reason I was leaving out auto-publishing was to "not include a deployment, on the principle that we shouldn't deploy something that doesn't have content" -- but it's not like this has content we don't want anyone to see. So I'll look into doing auto publishing, thanks for the references!